angular/src/app/workflows/workflow-list-single/workflow-list-single.component.ts

```diff
@@ -15,6 +16,8 @@ export class WorkflowListSingleComponent implements OnInit {
   workflowRunsResponse: WorkflowRunsResponse;
 
   newlyCreatedWorkflows?: any[];
+  pageToken = '';
```

Comment:
why initialize it? could improve this line as well https://github.com/DNAstack/ddap-frontend/pull/226/files#diff-fe720e93b99b0d6b7f66dacdac9cfe5bR61. It is not used anywhere else. Can it be removed?